apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -317,8 +317,9 @@ const FormTemplate: TemplateFunction<ButtonProps> = (props: ButtonProps) => html
     </form>
     <script>
         // Display a success message to the user when they submit the form
-        const form = document.querySelector('#testForm');
-        const formLog = document.querySelector('#formLog');
+        var form = document.querySelector('#testForm');
```

Comment:
I see you replaced a few consts with `var` to avoid the redeclaration error, could it be better to use `let` to signal that this is intended and not a old syntax leftover?

Reply:
I actually tried `let` first but you run into the same issue as with `const`, because the script is re-run from the beginning and you can't re-`let` a variable with the same name. `var` is different because you can use it to redeclare an existing variable, which is effectively what storybook is trying to do. 😞 

It is unfortunate but I thought this was the simplest solution. I could add a comment to explain the `var` usage though!

Reply:
Thanks Xander!